src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +174,283 @@
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props         :value
+                            :from-locked-amounts {:chain-id [1 10]}}}}})
+    (is (match? expected-db (:db (dispatch [event-id :chain-id]))))))
+
+(h/deftest-event :wallet/lock-from-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {:amount 10}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id :amount 10]))))))
+
+(h/deftest-event :wallet/clean-selected-token
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props        :value
+                            :token              "ETH"
+                            :token-display-name "ETH"
+                            :tx-type            :tx/collectible-erc-721}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/clean-selected-collectible
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props        :value
+                            :collectible        "ETH"
+                            :token-display-name "ETH"
+                            :amount             10}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/clean-suggested-routes
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send
+                     {:other-props                     :value
+                      :suggested-routes                ["1" "2"]
+                      :route                           "1"
+                      :amount                          10
+                      :from-values-by-chain            [{:chain-id 1} {:chain-id 10} {:chain-id 42161}]
+                      :to-values-by-chain              [{:chain-id 1} {:chain-id 10} {:chain-id 42161}]
+                      :sender-network-values           [:eth :arb1]
+                      :receiver-network-values         [:eth :arb1]
+                      :network-links                   [{:from-chain-id 1
+                                                         :to-chain-id   10
+                                                         :position-diff 1}]
+                      :loading-suggested-routes?       false
+                      :suggested-routes-call-timestamp ["1" "2"]}}}})
+    (is (match? expected-db (:db (dispatch [event-id]))))))
+
+(h/deftest-event :wallet/suggested-routes-error
```

Comment:
I think that big tests, where `expected-result` is a big structure would benefit from having more `(testing)` sections. If we split one big `match?` to few smaller and give them meaningful explanation describing what exactly we test, it will be easier for other developers to read the test and also to figure out the problem if it arises.

For example here we could have following `(testing)` parts:
- "sender network values reset to zero"
- "receiver network values reset to zero"
- "transaction error toast effect called"

wdyt?

Reply:
This event reset them all to zero without any condition. That's why I didn't use conditions and testings

Reply:
I have used this pattern in the next tests which will actually require it